ext/standard/string.c

```diff
@@ -2445,6 +2445,66 @@ PHP_FUNCTION(substr)
 }
 /* }}} */
 
+/* {{{ proto boolean str_begin(string search_value, string str)
+   Checks if str begins with search_value */
+PHP_FUNCTION(str_begin) {
+	zend_string *str, *search_value;
+	int i;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "SS", &search_value, &str) == FAILURE) 
+		RETURN_NULL();
+
+	for (i = 0; i < search_value->len; i++) 
+		if (str->val[i] != search_value->val[i]) 
```

Comment:
can you change the the interface to str_begin(string $haytstack, $needle): bool and str_ends(string $haystack, string $needle): bool for consistency with the functions with other string function like stristr, strpos, strrchr?